easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -233,6 +238,13 @@ def configure_step(self):
         else:
             make_ext = '.sys'
 
+        # patch Modules/parameter.f90 file to set compilation parameters
+        regex_para = []
+        for key in ['lmaxx', 'lqmax', 'npk', 'nsx', 'ntypx']:
```

Comment:
Since we have to have ntypx >= nsx it might be worth the effort to check that it is indeed the case.
I.e., start by picking up both from parameters.f90 then verify the requirement using cfg[key].
And you need to pick up the values from parameters in case someone sets only one of them in the easyconfig file.